course_catalog/views_test.py

```diff
@@ -215,21 +216,41 @@ def test_user_list_endpoint_get(client, is_public, is_author, user):
         assert resp.data.get("title") == another_user_list.title
 
 
+@pytest.mark.parametrize("is_public", [True, False])
+@pytest.mark.parametrize("is_staff", [True, False])
+@pytest.mark.parametrize("is_super", [True, False])
+@pytest.mark.parametrize("on_moira", [True, False])
 @pytest.mark.parametrize("is_anonymous", [True, False])
-def test_user_list_endpoint_create(client, is_anonymous, mock_user_list_index):
+def test_user_list_endpoint_create(  # pylint: disable=too-many-arguments
```

Comment:
Should expand the test for `PATCH` to also cover these conditions.

Reply:
Just added an `is_staff` condition in the following test.